### PR TITLE
Allow to overwrite `PHP_PHPCS_FILE_NAME` value

### PR DESCRIPTION
**Pull request by goetzc**
The PHP CodeSniffer linter itself allows different configuration file names:
https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#using-a-default-configuration-file

> Note: If multiple default configuration files are found, PHP_CodeSniffer will select one using the following order: `.phpcs.xml`, `phpcs.xml`, `.phpcs.xml.dist`, `phpcs.xml.dist`

This non-breaking change allow to select the name for the appropriate local file name.

## Proposed Changes

1. Allow to choose a different configuration file name.

### Reviewing Maintainer
- [x] Label as `breaking` if this is a large fundamental change
- [x] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

File: lib/linter.sh [hunk 1]
```diff
@@ -160,7 +160,7 @@ PERL_PERLCRITIC_OPTIONS="${PERL_PERLCRITIC_OPTIONS:-null}"
 # shellcheck disable=SC2034  # Variable is referenced indirectly
 PHP_BUILTIN_FILE_NAME="${PHP_CONFIG_FILE:-php.ini}"
 # shellcheck disable=SC2034  # Variable is referenced indirectly
-PHP_PHPCS_FILE_NAME="phpcs.xml"
+PHP_PHPCS_FILE_NAME="${PHP_PHPCS_FILE_NAME:-phpcs.xml}"
```
Comment (ferrarimarco) [inline lib/linter.sh:163]: Can you add the relevant entry in the environment vars table in the README?

**Review by ferrarimarco — CHANGES_REQUESTED** [review]
Thanks for this PR! A small doc update, and we shall be good to go.

File: lib/linter.sh (hunk 1 above)
Comment (goetzc) [reply lib/linter.sh:163]: Sure, that makes sense!

Comment (goetzc) [reply lib/linter.sh:163]: Done

**Review by ferrarimarco — APPROVED** [review]

**Review by zkoppert — APPROVED** [review]

**Pull request merged**